packages/pds/src/permissions/resources/blob-scope.ts

```diff
@@ -0,0 +1,67 @@
+import { NeRoArray, ParsedResourceScope, formatScope } from '../scope-syntax'
+
+type Accept = `${string}/${string}`
+function isAccept(value: string): value is Accept {
+  return (
+    value.includes('/') &&
+    !value.startsWith('/') &&
+    !value.endsWith('/') &&
+    !value.includes(' ')
+  )
+}
+
+const ALLOWED_PARAMS = Object.freeze(['accept'] as const)
+
+function matchAccept(accept: Accept, mime: string): boolean {
```

Comment:
We don't have to do it now, but we should eventually unify this logic with the accept validation of blobs (found in `packages/pds/src/actor-store/blob/transactor.ts`).